tools/resourcedocsgen/pkg/lookup.go

```diff
@@ -92,6 +92,7 @@ var CategoryLookup = map[string]PackageCategory{
 	"openstack":                           PackageCategoryCloud,
 	"opsgenie":                            PackageCategoryInfrastructure,
 	"pagerduty":                           PackageCategoryInfrastructure,
+	"pgedge":                              PackageCategoryDatabase,
```

Comment:
Instead of setting this in the override map, please just set `catagory/database` in your schemas `"keywords"` section with `tfbridge.ProviderInfo.Keywords`.